internal/metric/test_helpers.go

```diff
@@ -0,0 +1,66 @@
+package metric
```

Comment:
Is the end goal to have this eventually live in something like a `go.opentelemetry.io/otel/api/metric/metrictest` package?